brave/brave-core
# Load vpn urls from post-purchased app menu

fix https://github.com/brave/brave-browser/issues/18573

also some files that are not used by android are moved to mac/win specific
source list.


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch browser, enable vpn flag via brave://flags and close.
2. Launch browser with `--brave-vpn-test-credentials=aa:bb:cc:dd` to make vpn service in purchased state
3. Go to app menu -> Brave VPN -> click three entries (Send feedback and etc..) and check proper url is loaded in new tab.
